mappers/bluetooth_mapper/scheduler/scheduler.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/action_manager"
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/data_converter"
+	"github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/helper"
+)
+
+const (
+	MapperTopicPrefix     = "$ke/mappers/bluetooth-mapper/"
+	SchedulerResultSuffix = "/scheduler/result"
+)
+
+// Schedule is structure to define a schedule
+type Schedule struct {
+	//Enable indicates whether the scheduler is to be enabled or not
```

Comment:
specify defaults for fields like enable , frequency , limit etc.

Reply:
have now specified default for event frequency, the default for the other fields like enable and occurrence limit need not be given as they will take false and 0 by default if not given and the same has been handled in the code, thanks